src/platform/functions/index.js

```diff
@@ -194,3 +215,29 @@ exports.deleteOutdatedUserEvents = functions
     functions.logger.info('Deleting Successful', deletedEvents);
     res.send(deletedEvents);
   });
+
+exports.periodicRescraper = functions
+  .runWith({
+    timeoutSeconds: 60,
+    memory: '1GB',
```

Comment:
```suggestion
    timeoutSeconds: 120,
    memory: '2GB',
```
let's bump this up, cloud functions sometimes fail with lower ram and timeout

Reply:
cool! done!